### PR TITLE
New post: Electron's API Docs as Structured Data

### PR DESCRIPTION
**Pull request by zeke**
This is ready for eyes!

👀  @jlord @kevinsawicki

### CODE REVIEW

**Comment by zeke** [comment]
@Haacked too!

File: _posts/2016-09-27-api-docs-json-schema.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    main: true,
+    renderer: false
+  },
+  type: 'Class',
+  instanceName: 'win',
+  slug: 'browser-window',
+  websiteUrl: 'http://electron.atom.io/docs/api/browser-window',
+  repoUrl: 'https://github.com/electron/electron/blob/v1.4.0/docs/api/browser-window.md',
+  staticMethods: [],
+  instanceMethods: [],
```
Comment (Haacked) [inline _posts/2016-09-27-api-docs-json-schema.md]: I was a little confused that this class doesn't have any instance methods. But then below, I see that it does. Would it maybe help clarify things to add an ellipses in between the brackets to indicate there are instance methods, but we're omitting them for brevity?

File: _posts/2016-09-27-api-docs-json-schema.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+}
+```
+
+And here's an example of a method description, in this case the
+`apis.BrowserWindow.instanceMethods.setMaximumSize` instance method:
+
+```js
+{
+  name: 'setMaximumSize',
+  signature: '(width, height)',
```
Comment (Haacked) [inline _posts/2016-09-27-api-docs-json-schema.md]: Out of curiosity, what would the signature look like for a method that returns a value?

Comment (zeke) [inline _posts/2016-09-27-api-docs-json-schema.md]: @MarshallOfSound has a WIP for that: https://github.com/electron/electron/issues/7337

File: _posts/2016-09-27-api-docs-json-schema.md (hunk 1 above)
Comment (zeke) [inline _posts/2016-09-27-api-docs-json-schema.md]: Good call.

I was omitting them to keep it valid javascript. But I guess that doesn't matter here.

File: _posts/2016-09-27-api-docs-json-schema.md (hunk 2 above)
Comment (Haacked) [inline _posts/2016-09-27-api-docs-json-schema.md]: :cool:

**Pull request merged**